compiler/rustc_builtin_macros/src/global_allocator.rs

```diff
@@ -82,14 +81,24 @@ impl AllocFnFactory<'_, '_> {
         let result = self.call_allocator(method.name, args);
         let (output_ty, output_expr) = self.ret_ty(&method.output, result);
         let decl = self.cx.fn_decl(abi_args, ast::FnRetTy::Ty(output_ty));
-        let header = FnHeader { unsafety: Unsafe::Yes(self.span), ..FnHeader::default() };
+        let header = FnHeader {
+            unsafety: Unsafe::Yes(self.span),
+            ext: ast::Extern::from_abi(Some(ast::StrLit {
```

Comment:
`alloc/src/alloc.rs` uses `extern "Rust"` for `__rust_alloc` and friends. Seems like a mismatch?

Reply:
Indeed. I thought it used `extern "C"`. I will need to revert this change.